### ISSUE TITLE
Better keyboard less navigation

### ISSUE THREAD

**Issue by bbbbbgdn** [nodes-io/nodes-io#25]
I struggle navigating nodes using macbook.  Overall it feels much more keyboard-friendly, which is goof, but I'd wish for more clever UI decisions.

- I wish for pan gesture using touchpad, they implemented it in Cables. Two finger click pakes canvas to pan
- I'd like the code editor to close with some UI, not only ESC (which in my case is in touchbar)

...

**Comment by vorg** [comment]
Hi @bbbbbgdn! I agree with the editor close button. 

Could you explain more about panning? Is `[space] + touchpad press and drag` too cumbersome?

**Comment by bbbbbgdn** [comment]
yes, I think like it’s much more convenient when you can navigate interface and do basic things solely by using touchpad. 

like cables or more common figma, now you have general interface patterns that you expect by default

>

**Comment by dmnsgn** [comment]
> I'd like the code editor to close with some UI, not only ESC (which in my case is in touchbar)

At the moment, you can use `Code > Close Editor`.
